app/controllers/api/compact_index_controller.rb

```diff
@@ -30,7 +32,7 @@ def set_compact_index_cache_headers
   end
 
   def render_range(response_body)
-    headers['ETag'] = '"' << Digest::MD5.hexdigest(response_body) << '"'
+    headers['ETag'] = '"' << Digest::MD5.hexdigest(response_body) << '"'.dup
```

Comment:
Apparently unary plus is faster :sweat: Can you please change it back to `+`? Sorry.
Performance/UnfreezeString: Use unary plus to get an unfrozen string literal.
https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Performance/UnfreezeString

Reply:
yeah man! ;p I've seen this before, but I also agree that the syntax is not so fine at all